mltsp/time_series.py

```diff
@@ -0,0 +1,161 @@
+import copy
+import netCDF4
+import numpy as np
+import pandas as pd
+import xarray as xr
+from mltsp.cfg import config
+
+
+def _default_values_like(old_values, value=None, upper=None):
+    if value and upper:
+        raise ValueError("Only one of `value` or `upper` may be proivded.")
+    elif value is not None:
+        lower = value
+        upper = value
+    elif upper is not None:
+        lower = 0.
+    else:
+        raise ValueError("Either `value` or `upper` must be provided.")
+
+    new_values = copy.deepcopy(old_values)
+    if (isinstance(old_values, np.ndarray) and (old_values.ndim == 1
+                                                or 1 in old_values.shape)):
+        new_values[:] = np.linspace(lower, upper, len(new_values))
+    else:
+        for new_array in new_values:
+            new_array[:] = np.linspace(lower, upper, len(new_array))
+
+    return new_values
+
+
+def make_array(x):
+    try:
+        x = np.asfarray(x).squeeze()
```

Comment:
May introduce a copy--is that intended?  And why is the failure mode here to simply return the original?